packages/ui/src/showcases/DepositAndBorrowPage.stories.tsx

```diff
@@ -0,0 +1,404 @@
+                  </BoxLayout>
+                </CardLayout>
+              </GridLayout.Column>
+              <GridLayout.Column span={6}>
+                <CardLayout>
+                  <BoxLayout gap={2}>
+                    <StackLayout gap={2.815}>
+                      <Text as="span" color="oxfordBlue" type="heading 2">
+                        Your borrow
+                      </Text>
+                      <Text type="body 5">Nothing borrowed yet</Text>
+                    </StackLayout>
+                  </BoxLayout>
+                </CardLayout>
+              </GridLayout.Column>
+              <GridLayout.Column span={12}>
+                <CardLayout>
+                  <BoxLayout gap={8}>
+                    <CenterLayout>
+                      <Text as="span" color="oxfordBlue" type="heading 2">
+                        Please, connect your wallet
+                      </Text>
+                      <Text color="oxfordBlue" type="body 5">
+                        Please, connect your wallet to see your deposits and
+                        borrowings
+                      </Text>
+                      <Button
+                        appearance="primary"
+                        size="medium"
+                        variant="medium"
+                      >
+                        Connect wallet
+                      </Button>
+                    </CenterLayout>
+                  </BoxLayout>
+                </CardLayout>
+              </GridLayout.Column>
+            </GridLayout>
+          </StackLayout>
+        </PageLayout.Section>
+      </PageLayout>
+    </BrowserRouter>
+  )
+}
+
+export const Connected = args => {
+  const nav = usePageLayout()
+  const { setHeader } = nav
+
+  useEffect(() => {
+    setHeader(getHeader())
+  }, [setHeader])
+
+  const [isModalOpen, setIsModalOpen] = useState<
+    'borrow' | 'deposit' | 'withdraw' | 'repay' | undefined
+  >(undefined)
+  const closeModal = () => setIsModalOpen(undefined)
+  const isDepositEmpty = args.depositRows === 0
+  const isBorrowEmpty = args.borrowRows === 0
+
+  const yourDepositColumns = {
+    assets: {
+      header: 'Assets',
+      Cell: ({ data: { assets } }: CellProps) => <span>{assets}</span>,
+      alignText: 'left',
+    },
+    balance: {
+      header: 'Balance',
+      Cell: ({ data: { balance } }: CellProps) => <span>{balance}</span>,
+      alignText: 'left',
+    },
+    APY: {
+      header: 'APY',
+      Cell: ({ data: { APY } }: CellProps) => <span>{APY}</span>,
+      alignText: 'left',
+    },
+
+    collateral: {
+      header: 'Collateral',
+      Cell: () => <Toggle />,
+      alignText: 'center',
+    },
+    action: {
+      header: '',
+      Cell: () => (
+        <Button
+          appearance="secondary"
+          variant="thin"
+          size="small"
+          onClick={() => setIsModalOpen('withdraw')}
+        >
+          Withdraw
+        </Button>
+      ),
+      width: 100,
```

Comment:
I think we said that this should be 160? is it working now properly?

Reply:
@aleksaToljic  Problem was with button, when I add it a class "u-full-width" it works as expected, no issues. But I also see that button should be 120px width, not sure how I found 100, or 160 :disappointed: